### DISCUSSION TITLE
关于Github登录，希望能够配置代理地址

### DISCUSSION THREAD

**Discussion by ZvonimirSun** [walinejs/waline#166] [Ideas]
在国内服务器部署Waline，通过Github登陆时，有时后台获取token会响应超时，希望能够配置使用代理地址。

**Comment by lizheming** [comment]
这个之后会修改社交登录的逻辑，会将社交登录单独抽成一个官方服务，所有的 Waline 都调该服务来处理。

**Reply by ZvonimirSun** [reply]
好的，了解了
